sources/packages/backend/apps/api/src/route-controllers/dynamic-form/dynamic-form.controller.ts

```diff
@@ -23,12 +28,46 @@ export class DynamicFormController extends BaseController {
     super();
   }
 
+  /**
+   * List all form definitions that contains the tag 'common' ordered by title.
+   * @returns list of form definitions.
+   */
+  @Roles(Role.AESTFormEditor)
   @Get()
-  async list(): Promise<any> {
-    return this.formService.list();
+  async list(): Promise<FormsAPIOutDTO> {
+    const allForms = await this.formService.list();
+    return {
+      forms: allForms.map((form) => ({
+        name: form.name,
+        title: form.title,
+      })),
+    };
   }
```

Comment:
Following our client based seperation, these endpoints must be under *.aest.controller

Reply:
I will move this one that was already misplaced, and the new ones.

Reply:
Controller methods moved.